modules/tinymce/src/themes/silver/main/ts/ui/menus/item/structure/ItemSlices.ts

```diff
@@ -88,6 +90,34 @@ const renderDownwardsCaret = (icons: IconProvider): AlloySpec => ({
   }
 });
 
+const renderContainer = (container: Menu.CardContainer, components: Array<AlloySpec>): AlloySpec => ({
+  dom: {
+    tag: 'div',
+    classes: [
+      ItemClasses.containerClass
+    ].concat([
+      container.direction === 'vertical' ? ItemClasses.containerColumnClass : ItemClasses.containerRowClass
+    ]).concat([
+      container.align === 'left' ? ItemClasses.containerAlignLeftClass : ItemClasses.containerAlignRightClass
+    ]).concat([
+      container.valign === 'top' ? ItemClasses.containerValignTopClass :
```

Comment:
This is nested ternaries, which are normally banned. Can this be rewritten to remove the nested ternary?

- [ ] Rewrite to avoid nested ternaries if possible.